pom.xml

```diff
@@ -119,11 +119,11 @@ under the License.
 			<scope>test</scope>
 		</dependency>
 
-		<dependency>
-			<groupId>org.testcontainers</groupId>
-			<artifactId>junit-jupiter</artifactId>
-			<scope>test</scope>
-		</dependency>
+        <dependency>
+	       <groupId>org.testcontainers</groupId>
+	       <artifactId>junit-jupiter</artifactId>
+	       <scope>test</scope>
+        </dependency>
```

Comment:
I think there is no need to modify the indentation symbol here.

Reply:
fix